python/lsst/meas/base/wrappers.py

```diff
@@ -119,8 +125,7 @@ def wrapAlgorithm(Base, AlgClass, factory, executionOrder, name=None, Control=No
                                If apCorrList is non-empty then shouldApCorr is ignored.
                                If non-empty and doRegister is True then the names are added to the set
                                retrieved by getApCorrNameSet
-
-
+    @param[in] hasLogName      Plugin supports a logName as a constructor argument
```

Comment:
Removed extra blank line.  Not sure what the standard is, but should be in a separate commit.